nisystemlink/clients/assetmanagement/_asset_management_client.py

```diff
@@ -122,3 +122,21 @@ def delete_assets(self, ids: List[str]) -> models.DeleteAssetsResponse:
             ApiException: If unable to communicate with the asset management service or if there are invalid arguments.
         """
         ...
+
+    @post("assets/{assetId}/file", args=[Path("assetId"), Field("fileIds")])
+    def link_files(
+        self, asset_id: str, file_ids: List[str]
+    ) -> models.LinkFilesPartialSuccessResponse:
```

Comment:
If this succeeds fully it will return None so we should reflect that in the type hint.
```suggestion
    ) -> Optional[models.LinkFilesPartialSuccessResponse]:
```

Reply:
updated as per suggestion